### PR TITLE
config(perf-regression-throughput): Added qa@scylladb.com to email_recipients

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
Since it's under our responsibility to inspect the master performance tests, I added the entire qa team to the email recipients

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: tests/perf-regression.100threads.30M-keys-i3.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ append_scylla_args: '--blocked-reactor-notify-ms 4'
 
 # email definitions to send regression report
 send_email: true
-email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com']
+email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com', 'balalis@scylladb.com']
```
Comment (bentsi) [inline tests/perf-regression.100threads.30M-keys-i3.yaml]: @roydahan why isn't it sent to qa?

Comment (roydahan) [reply tests/perf-regression.100threads.30M-keys-i3.yaml]: change to QA

Comment (ShlomiBalalis) [reply tests/perf-regression.100threads.30M-keys-i3.yaml]: Done

**Comment by fruch** [comment]
@ShlomiBalalis I think the idea to put there only `qa@scylladb.com` all of the 3 others are part of it

File: tests/perf-regression.100threads.30M-keys-i3.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ append_scylla_args: '--blocked-reactor-notify-ms 4'
 
 # email definitions to send regression report
 send_email: true
-email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com']
+email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com', 'qa@scylladb.com']
```
Comment (bentsi) [inline tests/perf-regression.100threads.30M-keys-i3.yaml]: should be `email_recipients: ['qa@scylladb.com']`

Comment (bentsi) [reply tests/perf-regression.100threads.30M-keys-i3.yaml]: all of the mentioned persons are part of the QA group

Comment (ShlomiBalalis) [reply tests/perf-regression.100threads.30M-keys-i3.yaml]: I figured that maybe that could ruin some rules the other recipients already have, but sure

**Pull request merged**